frontend/src/components/enroll/index.jsx

```diff
@@ -8,15 +8,12 @@ import SingleCarousel from "../SingleCarousel";
 class Enroll extends Component {
   constructor(props) {
     super(props);
-
     this.state = {
       text: null,
       loadCounter: 0,
       iframeHeight: 1300
     };
-  }
 
-  componentWillMount() {
```

Comment:
I believe you want to use one of the lifecycle methods for this maybe `componentDidMount` so that you don't hold up showing things while the fetch call happens